bofhound/ad/adds.py

```diff
@@ -592,13 +766,12 @@ def parse_acl(self, entry:BloodHoundObject):
                     # Since 4.0
                     # Key credential link property write rights
                     if entry._entry_type.lower() in ['user', 'computer'] and ace_object.acedata.has_flag(ACCESS_ALLOWED_OBJECT_ACE.ACE_OBJECT_TYPE_PRESENT) \
-                    and ace_object.acedata.get_object_type().lower() == '5b47d60f-6090-40b2-9f37-2a4de88f3063' \
```

Comment:
I prefer not keeping "hardcoded" values. The fact is that if you collect schema configuration, you should not have problem, because nowadays :

`self.ObjectTypeGuidMap['ms-ds-key-credential-link'] == '5b47d60f-6090-40b2-9f37-2a4de88f3063'`

if you collected well all schemaguid values

We can keep hardcoded value if you want but my opinion is to remove it